src/azure-cli/azure/cli/command_modules/appconfig/feature.py

```diff
@@ -49,7 +49,7 @@ def set_feature(cmd,
                 auth_mode="key",
                 endpoint=None):
     if key is None and feature is None:
-        raise RequiredArgumentMissingError("Please provide either `--key` or `--feature` value.")
+        raise CLIErrors.RequiredArgumentMissingError("Please provide either `--key` or `--feature` value.")
```

Comment:
May I ask why do you need to add the prefix `CLIErrors.` to these error type?

Reply:
@zhoxing-ms This was done because some of the errors have names that conflict with the [exceptions ](https://github.com/Azure/azure-cli/pull/24400/files#:~:text=(HttpResponseError%2C-,from%20azure.core.exceptions%20import%20(HttpResponseError%2C,ResourceModifiedError),-from%20._constants )imported from `azure.core.exceptions`.